### PR TITLE
Ops-908 Upgrade jQuery from 2.2's Security Vulnerability

### PR DESCRIPTION
**Pull request by jonhill13**
**Do Not Merge**

- [x] Will be creating a PR off of master and will be downmerging. Just leaving this PR here until we complete those changes.

**Description:**

This PR removes the security vulnerability by migrating jQuery from 2.2 to 3.4.1 with minimal impact to the user.

**Technical details:**

* upgrade to jQuery 3.4.1

* upgrade bootstrap to 4.0.0

* we will not update main.css since there is custom css mixed with minified versions of npm packages

* User's will not keep their side menu on the right when scrolling.

* Recommend merging the API site with the React App or a refactor.

**Requirements for PR merge:**

1. [N/A] Unit & integration tests updated
2. [N/A] API documentation updated
3. [ ] Necessary PR reviewers:
    - [ ] Backend
    - [x] Frontend @ebdabbs 
4. [N/A] Matview impact assessment completed
5. [N/A] Frontend impact assessment completed
6. [ ] Data validation completed
7. [N/A] Appropriate Operations ticket(s) created
8. [ ] Jira Ticket [OPS-908](https://federal-spending-transparency.atlassian.net/browse/OPS-908):
    - [ ] Link to this Pull-Request
    - [N/A] Performance evaluation of affected (API | Script | Download)
    - [ ] Before / After data comparison

**Area for explaining above N/A when needed:**
```
This PR affects the USASpending API website only. Validation of the API website is necessary.
```

### CODE REVIEW

File: usaspending_api/static_doc_files/js/jquery-2.2.0.min.js [hunk 1]
```diff
@@ -1,4 +1,2 @@
-/*! jQuery v2.2.0 | (c) jQuery Foundation | jquery.org/license */
```
Comment (tony-sappe) [inline usaspending_api/static_doc_files/js/jquery-2.2.0.min.js]: Shouldn't the newer version of jQuery live in `jquery-3.4.1.min.js`?

Would need to update the filename src links in /usaspending_api/templates/index.html and /usaspending_api/templates/plate.html

Comment (jonhill13) [reply usaspending_api/static_doc_files/js/jquery-2.2.0.min.js]: Concur

**Comment by tony-sappe** [comment]
@jonhill13 any concerns with Bootstrap 4 not supporting jQuery 3?
image

**Comment by jonhill13** [comment]
@tony-sappe 
From the [Bootstrap 4 Docs](https://getbootstrap.com/docs/4.0/migration/#breaking)
`Bootstrap still requires jQuery 1.9.1 or higher, but you’re advised to use version 3.x since v3.x’s supported browsers are the ones Bootstrap supports plus v3.x has some security fixes.`

This issue you see above is already in prod.

**Comment by jonhill13** [comment]
This PR is replaced by this one to master. Then we will downmerge to staging and then dev.
https://github.com/fedspendingtransparency/usaspending-api/pull/2186

**Pull request closed without merging**